### ISSUE TITLE
转成https上传应该在哪进行配置

### ISSUE THREAD

**Issue by Yohansun** [Macrow/rails_kindeditor#90]
`Mixed Content: The page at “https”was loaded over HTTPS, but requested an insecure form action 'http://xxxxx/kindeditor/upload?dir=image'. This request has been blocked; the content must be served over HTTPS.`
富文本编译器因我们改成了https 格式报错，想问什么地方可以进行配置吗

**Comment by Macrow** [comment]
rails_kindeditor 里面只是配置了相对url，并没有指定http或者https，

你用rake routes看看路由，看看到底哪里出了问题

**Comment by KLoong** [comment]
我也出现了相同的问题。

**Comment by Yohansun** [comment]
我是对域名做了两个配置 http和https

**Comment by KLoong** [comment]
我是在当前的https的域名下面，但是kindeditor的表单生成的是http的链接，这样提交就会报错了。

**Comment by Yohansun** [comment]
是的 我们当时看源代码来着 但是好像不了了之了 我们是后台用 后台就增配了http可以打开 https也可以打开 当有表单的时候 默认打开http连接 就不报错了

**Comment by clownf** [comment]
有可能是nginx配置问题
